harness/determined/cli/workspace.py

```diff
@@ -111,39 +112,34 @@ def list_workspaces(args: Namespace) -> None:
 @authentication.required
 def list_workspace_projects(args: Namespace) -> None:
     sess = cli.setup_session(args)
-    w = workspace_by_name(sess, args.workspace_name)
-    orderArg = bindings.v1OrderBy[args.order_by.upper()]
-    sortArg = bindings.v1GetWorkspaceProjectsRequestSortBy[args.sort_by.upper()]
-    internal_offset = args.offset if ("offset" in args and args.offset) else 0
-    limit = args.limit if "limit" in args else 200
-    all_projects: List[bindings.v1Project] = []
-
-    while True:
-        projects = bindings.get_GetWorkspaceProjects(
-            sess,
-            id=w.id,
-            limit=limit,
-            offset=internal_offset,
-            orderBy=orderArg,
-            sortBy=sortArg,
-        ).projects
-        all_projects += projects
-        internal_offset += len(projects)
-        if ("offset" in args and args.offset) or len(projects) < limit:
-            break
+    all_projects = workspace.Workspace(
+        session=sess, workspace_name=args.workspace_name
+    ).list_projects()
+
+    sort_key = args.sort_by.lower()
+    sort_order = args.order_by.lower()
+    offset = args.offset or 0
```

Comment:
can we put these defaults in `args_description`?

Reply:
A good idea that someone already beat you to: https://github.com/determined-ai/determined/blob/main/harness/determined/cli/workspace.py#L378-L391

Reply:
Looks like
```
% det -m latest-main.determined.ai:8080 w list-projects --help
usage: det workspace list-projects [-h] [--sort-by {id,name}] [--order-by {asc,desc}] [--limit LIMIT] [--offset OFFSET] [--json] workspace_name

positional arguments:
  workspace_name        name of the workspace

optional arguments:
  -h, --help            show this help message and exit
  --sort-by {id,name}   sort workspaces by the given field (default: id)
  --order-by {asc,desc}
                        order workspaces in either ascending or descending order (default: asc)
  --limit LIMIT         Maximum items per page of results (default: 200)
  --offset OFFSET       Number of items to skip before starting page of results (default: None)
  --json                print as JSON (default: False)
```

Reply:
so we don't need these defaults anymore right? i meant the defaults should be set in `args_description`, not just the help text

Reply:
ohh. I'm here only days later understanding what you meant. _These_ defaults meant the defaults that I've added inline. Since they were already there before this PR, I didn't even consider what I did when refactoring it.

Yes. Agreed. Thanks. Getting rid of them.

Reply:
Also didn't need to lowercase the sort args, since that's also enforced by the argparser.